bcgov/GDX-Analytics
# updated lock file and bumped express-4.19.2.tgz & semver-6.3.1.tgz

This PR updates the package-lock.json with a fresh npm install and bumps express-4.19.2.tgz & semver-6.3.1.tgz, solving the remaining dependabot alerts for the examples/react_tracker.

Testing instructions:

- clone the repo and switch to dependant branch
```
git clone https://github.com/bcgov/GDX-Analytics.git
cd GDX-Analytics/
git checkout GDXDSD-6851-merge-react-tracker-branches
```
- install react and related packages globally
```
npm install -g react react-dom react-router-dom
```
- install the dependencies listed in the package-lock.json
```
cd examples/react_tracker/
npm ci
```
- start a development server
```
npm start
```
check if sample react.js page is running by browsing to http://localhost:3000/
